packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

```diff
@@ -258,6 +268,9 @@ export const AddNewTrustline: FC = () => {
   }
```

Comment:
no bad_request for if (isParamsMissing) { ?

Reply:
Done on line 128.

Reply:
I would move it here since, for now isParamsMissing is only for limitAmount parsing, but this may change. We should trigger the error if `paramsMissing` is true

Reply:
for example, next MR will embed flags, memos, etc